cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -234,6 +234,9 @@ func createDaisyInflater(args ImportArguments, fileInspector imagefile.Inspector
 		return nil, err
 	}
 
+	if args.UefiCompatible {
+		daisyUtils.UpdateToUEFICompatible(wf)
```

Comment:
Maybe use addFeatureToDisk, similarly to how WINDOWS is added? This is fine as well but it does more than needed here. daisyUtils.UpdateToUEFICompatible(wf) was useful for monolithic Daisy workflows. With these fine grained workflows we can avoid it.

Reply:
sgtm; updated.